test/specs/rules/validate-indentation.js

```diff
@@ -156,6 +156,23 @@ describe('rules/validate-indentation', function() {
         ]);
     });
 
+    it('should not report errors for using tabs when validateIndentation is a number', function() {
+        checker.configure({ validateIndentation: 4 });
```

Comment:
I'm not sure I understand what you mean, but all's I'm suggesting is that jscs figures out what indentation is intended in the document. Maybe it looks for what comes first: tabs or spaces, and then when it sees that, it assumes the rest of the document should use that kind of indentation. I just want to clarify: I want consistent indentation, but I also want the rule to adjust to whatever kind of indentation seems to be the main indentation of the file.